contracts/interfaces/ITwabRewards.sol

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.6;
+
+import "@pooltogether/v4-core/contracts/interfaces/ITicket.sol";
+
+/**
+ * @title  PoolTogether V4 ITwabRewards
+ * @author PoolTogether Inc Team
+ * @notice TwabRewards contract interface.
+ */
+interface ITwabRewards {
+    /**
+        @notice Struct to keep track of each epoch's settings.
+        @param id Epoch id to keep track of each epoch
+        @param startTimestamp Timestamp at which the epoch starts
+        @param duration Duration of one epoch in seconds
+     */
+    struct Epoch {
+        uint32 id;
+        uint32 startTimestamp;
+        uint32 duration;
+    }
```

Comment:
✂️ 

You can cut the struct- notice how the id + duration are duplicated elsewhere.  The epoch id is already known, and the duration is already in the promotion struct.

Really, I think the user only needs:

'getCurrentEpochId()' - returns the current epoch id

The epoch duration is already known, and the start time is easily calculated based on promo + epoch id

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/8/commits/423f188e97b1edd4ff89b6b5621bde3f656b0fbc